src/Bicep.Core/Syntax/ParameterDeclarationSyntax.cs

```diff
@@ -1,95 +1,104 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
I didn't change anything it this file, I think when I was debugging in VS, this file opened and I hit to unify line endings.